libs/sdk-core/src/models.rs

```diff
@@ -1288,6 +1288,8 @@ pub struct SwapInfo {
     ///
     /// Swaps created after dynamic fees were introduced always have this field set.
     pub channel_opening_fees: Option<OpeningFeeParams>,
+    /// The block height when the swap was confirmed.
+    pub confirmed_at: u32,
```

Comment:
Would it be better for it to be an Option<u32> and nullable in the table?

Reply:
Yes, makes sense. Donw